pkg/chartmuseum/server/multitenant/handlers.go

```diff
@@ -82,7 +83,18 @@ type (
 )
 
 func (server *MultiTenantServer) getWelcomePageHandler(c *gin.Context) {
-	c.Data(200, "text/html", welcomePageHTML)
+	if server.WebTemplatePath != "" {
+		c.HTML(http.StatusOK, "index.html", nil)
+	} else {
+		c.Data(http.StatusOK, "text/html", welcomePageHTML)
+	}
+}
+
+func (server *MultiTenantServer) getStaticFilesHandler(c *gin.Context) {
```

Comment:
same here, it be possible to update the unit tests to cover this new handler?

Reply:
thanks for pointing out, ofc will do!

Reply:
Done